### ISSUE TITLE
Z Index

### ISSUE THREAD

**Issue by solidoussnk** [d-oliveros/ngSticky#122]
Is there a way to configure the z-index? I have a fly out menu which shows under the sticky.

**Comment by tucq88** [comment]
IMO, you should use a custom class for the sticky (when it stuck) and set right `z-index` number for it to make it works with your menu.

**Comment by circlingthesun** [comment]
@tucq88, element level styles override class based styles so thats not an option.

**Comment by solidoussnk** [comment]
Actually I was able to override using !important, thanks.

**Comment by circlingthesun** [comment]
Right, I forgot about !important. I just forked the project and implemented a sticky-z-index attribute. I would a PR be worthwhile?

**Comment by tucq88** [comment]
@circlingthesun :+1:  I'm not sure about the z-index PR, because it could be done by using a custom class via options so I think we should not set it as default :D IMO, the more flexibility, the better :smile:
